README.md

```diff
@@ -68,7 +68,8 @@ Commands:
   help [command]             display help for command
 ```
 
-### Create Process Command
+
+### Crete Process Command
```

Comment:
typo

Reply:
must have accidentally deleted, as this line should have not been changed, thanks for flagging, as always!